ihmc-high-level-behaviors/src/libgdx/java/us/ihmc/rdx/ui/affordances/RDXArmManager.java

```diff
@@ -380,22 +380,15 @@ public void executeDesiredArmCommand(RobotSide robotSide)
             RDXBaseUI.pushNotification("Commanding arm jointspace trajectory...");
             communicationHelper.publishToController(armTrajectoryMessage);
          }
-         case TASKSPACE ->
```

Comment:
removed taskspace without joints, as this is not the right way to control the arm and it's never worked.
As an external user, I want to either control the arm in jointspace or taskspace, with or without impedance control. The fact that the taskspace control is using a hybrid message shouldn't concern the user